ControlMeasurements/Controllers/HomeController.cs

```diff
@@ -1,16 +1,38 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using ControlMeasurements.Data;
+using ControlMeasurements.ViewModels;
 using Microsoft.AspNetCore.Mvc;
+using System.Linq;
 
 namespace ControlMeasurements.Controllers
 {
     public class HomeController : Controller
     {
+        private MeasurementsContext _context;
+
+        public HomeController(MeasurementsContext context)
+        {
+            _context = context;
+        }
+
         public IActionResult Index()
         {
-            return View();
+            var measurements = _context.Measurements;
+
+            var result = new HomeIndexViewModel
+            {
+                WaterMeasurements = measurements
+                                    .Where(x => x.MeasurementType == MeasurementType.Water).Take(3).OrderByDescending(x=>x)
```

Comment:
Najpierw bierzesz 3 wyniki a potem je sortujesz? Chyba powinno być na odwrót :)
Plus musisz mu wskazać konkretnie po ktorym polu chcesz sortować, czyli na przykład: x => x.Value.
Jesli chcesz sortowac po dacie, to musiałbyś dodać pole z datą do modelu Measurement :)

Reply:
No tak logiczniej by było ;) Ale to też dzialalo;)

Reply:
Hmmm ok :D